### PR TITLE
Launch Screen + GIF + Fonts

### PR DESCRIPTION
**Pull request by albertomota-codes**
Also:
StackView over scroll view
Fix contraints issue

### CODE REVIEW

**Comment by albertomota-codes** [comment]
ezgif com-video-to-gif (1)

File: Ventivader/Ventivader/Utilities/FontsVentivader.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//  FontsVentivader.swift
+//  Ventivader
+//
+//  Created by Al on 11/04/20.
+//  Copyright © 2020 calculator. All rights reserved.
+//
+
+import UIKit
+
+class FontsVentivader {
```
Comment (cskashyap) [inline Ventivader/Ventivader/Utilities/FontsVentivader.swift:11]: Can we all this VentivaderFonts?

Comment (albertomota-codes) [reply Ventivader/Ventivader/Utilities/FontsVentivader.swift]: Done

**Comment by cskashyap** [comment]
Looks Great! Please go ahead and merge it.

**Pull request merged**